### PR TITLE
Update dev_solutions.md

### PR DESCRIPTION
**Pull request by gprasath**
-Provided links 
-Added chmod command in quotes.
-Modified few sentences

### CODE REVIEW

**Comment by gprasath** [comment]
@mahebbar , can you please review this

File: docs/dev_solutions.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following commands help you install Docker on Ubuntu version 16.04 (64 bit).
 
 ## Add iSCSI Support
 
-On your Ubuntu host, install open-iscsi package. OpenEBS uses iSCSI to connect to the block volumes. 
+On your Ubuntu host, install open-iscsi package. OpenEBS create iSCSI based block volumes that offers the persistant storage.
```
Comment (mahebbar) [inline docs/dev_solutions.md]: creates

File: docs/dev_solutions.md [hunk 2]
```diff
@@ -159,19 +161,19 @@ The following snippet of the openebs-operator.yaml -\> maya-apiserver section sh
               value: "1"
     ---
 
-Add OpenEBS related storage classes, that can then be used by developers and applications using the following command.
+Add OpenEBS related storage classes, that can then be used by developers and applications by running following command.
```
Comment (mahebbar) [inline docs/dev_solutions.md]: by running the following command.

Comment (gprasath) [reply docs/dev_solutions.md]: Modified

File: docs/dev_solutions.md (hunk 1 above)
Comment (gprasath) [reply docs/dev_solutions.md]: Modified

**Pull request closed without merging**